pages/home/index.tsx

```diff
@@ -27,25 +27,32 @@ export default function Home() {
     setTasks(tasks ? tasks : []);
   }, []);
 
-  useEffect(() => {
-    addEventListener("keypress", ({ key }: any) => {
-      if (key == "Enter") {
-        handleClick();
-      }
-    });
-  }, []);
+
 
   const handleClick = () => {
     let tasks: any = JSON.parse(
       // @ts-ignore
       localStorage.getItem("tasks") ? localStorage.getItem("tasks") : []
     );
-    // @ts-ignore
+
+    try {
+       // @ts-ignore
     if (inputRef.current.value && !isFound(tasks, inputRef.current.value)) {
       // @ts-ignore
       tasks.push(inputRef.current.value);
+    };
+    } catch (error) {
+      console.log(error);
     }
```

Comment:
remove this try and catch

```suggestion
       // @ts-ignore
    if (inputRef.current.value && !isFound(tasks, inputRef.current.value)) {
      // @ts-ignore
      tasks.push(inputRef.current.value);
    };
```